site/_posts/2019-10-08-deactivating-reactivating-deleting-user-fusionauth-python.md

```diff
@@ -205,72 +214,72 @@ Save and execute the `reactivate.py` application on your server. Upon success, y
 }
 ```
 
-Refresh the **Users** page in your browser and you'll now notice that the lock icon has disappeared beside our test user and the account has been reactivated.
+Refresh the "Users" page in your browser and you'll now notice that the lock icon has disappeared beside our test user and the account has been reactivated.
 
-{% include _image.liquid src="/assets/img/blogs/terry-example-unlocked.png" alt="Terry is now unlocked" class="img-fluid mb-4 mt-2 mx-auto d-block" figure=false %}
+{% include _image.liquid src="/assets/img/blogs/terry-example-unlocked.png" alt="Terry is now unlocked" class="img-fluid" figure=false %}
 
-## How to Delete A User With FusionAuth-Client
+## How to delete a user with FusionAuth Python client library
 
-For our next trick, we will be deleting a test user from the server. If you filled out a lot of profile information on your regular test user, you might want to create an empty test user for this tutorial. Just be sure to grab their UserID from the **Actions-->Manage** section of FusionAuth.
+For our next trick, we will be deleting a test user from the server. If you filled out a lot of profile information on your regular test user, you might want to create an empty test user for this tutorial. Just be sure to grab their "User Id" from the "Manage User" section of FusionAuth.
```

Comment:
Why would you delete a user vs deactivate them? Did that get covered above (might have missed it).